schemas/json/README.md

```diff
@@ -96,17 +107,15 @@ Second, the [lexical space] of XSD values is much larger than the lexical space
 For example, both `1` and `true` are valid lexical representations of a true value.
 In contrast, JSON recognizes only `true`.
 
+
 [lexical space]: https://www.w3.org/TR/xmlschema11-2/#lexical-space
 
-Since the semantic of the value is defined outside the model, and might depend directly on the lexical representation, we must represent these values as strings.
-For example, we might define through the attribute [Property/semanticId] that a [Property/value], given as a boolean, distinguishes between `1` and `true` where `1` has a completely different meaning from `true`.
-This distinction would have been lost if we simply serialized the value to a JSON boolean.
+In many cases the semantics of a [Property] or any other [SubmodelElement] is also restricting its value range in case of numerical values. However, when just looking at the [Property/valueType] this information is not given`and the value range seems larger than it actually is allowed to be.
```

Comment:
Revisiting this sentence. I'd recommend you to remove it, or move it to a different section. The problem is that JSON numbers and boolean _restrict_ the space _more_ than a concept descript might allow for, not less!

For example, if a concept is a 64-bit hash ID, it would make sense to represent it as `xs:long`. Since JSON numbers are used, it must be represented as `xs:string`. 

Or a concept some sensor value. If infinity or NaN values are possible to be sent from the sensor, the value type again has to be `xs:string` and the value represented as JSON string, not `xs:double`, as JSON numbers do not allow for infinity and NaN.


I'd recommend you to make examples in the readme as this behavior is quite opaque, and not self-evident at all. I suppose picking a small number, large number, a NaN and -0 would suffice, and show how Property would look like in those cases.

Reply:
This is not correct, since the Property/ValueType is derived from the concept description und not vice versa.
It is always a two-step mapping:
- Mapping from Concept Descirption to xsd
- Mapping from  XSD to target language

For JSON we do habe a mapping table that is the master for the mapping. So if changes are needed the change requests need to be addressed to this mapping table in part 2.

Reply:
Please re-visit this part of my comment:
```
For example, if a concept is a 64-bit hash ID, it would make sense to represent it as xs:long. Since JSON numbers are used, it must be however represented as xs:string.
```

So CD -> xs:long. Xs:long does not fit in JSON number (2^63-1 > 2^52). Therefore xs:long's are represented as JSON strings? Or only if the concrete value is larger than 2^52?

Reply:
As I stated: this is a request to the mapping table not to the serialization rule themselves that just states that this table is used.

Reply:
additionallty added comment in issue on source of mapping tabel, see below